tools/src/keygen/Dump.hs

```diff
@@ -99,7 +100,7 @@ dumpKeyfiles (dir, pat) dlgIssuers richs poors = do
         dumpDlgIssuerSecret (dlgIssuersDir </> patternize i) sk
     forM_ (enumerate richs) $ \(i, richSecrets) ->
         dumpRichSecrets (richDir </> patternize i) richSecrets
-    forM_ (enumerate poors) $ \(i, hdwSk) ->
+    forM_ (enumerate . map poorSecretToEncKey $ poors) $ \(i, hdwSk) ->
```

Comment:
I find it more natural to pass `PoorSecret` to `dumpPoorSecret` function and make this conversion there.